infra/charts/feast-feature-server/values.yaml

```diff
@@ -7,14 +7,26 @@ replicaCount: 1
 image:
   # image.repository -- Docker image for Feature Server repository
   repository: feastdev/feature-server
-  pullPolicy: IfNotPresent
+  pullPolicy: Always
   # image.tag -- The Docker image tag (can be overwritten if custom feature server deps are needed for on demand transforms)
   tag: 0.39.0
 
 imagePullSecrets: []
 nameOverride: ""
 fullnameOverride: ""
 
+metrics:
+  enabled: false
+  otelCollector:
+    endpoint: ""  # sample endpoint: "otel-collector.default.svc.cluster.local:4317"
+    port: 4317
+
+otel_service:
```

Comment:
`otel_service` is duplicated and probably not even in use and can be removed

Reply:
+1